### DISCUSSION TITLE
Have UP and DOWN arrow always switch between columns

### DISCUSSION THREAD

**Discussion by ZenaMel** [opentoonz/opentoonz#6731] [Ideas]
### Is your feature request related to a problem? Please describe

Right now, when selecting a column using the mouse, pressing arrow up/down switches between columns.
Once I start drawing on any column though, the up/down arrow keys seek through the frames just like the left/right arrow keys do, until I select any column with the mouse again.

### Describe the solution you'd like

By default, have it so that the up and down arrow keys switch between columns and left and right arrow keys seek through the frames even if the user starts using a drawing tool. In other words: have that be the behaviour at all times.

Alternatively/additionally, provide an option to configure what the up/down arrow keys do (maybe that's already in, couldn't find it myself in v1.6)

**Comment by flurick** [comment]
Works as suggested when using the ComboViewer window instead of the Viewer+Toolbar, so I guess this is a bug in the Toolbar getting the keyboard focus.
